.vscode/settings.json

```diff
@@ -4,4 +4,5 @@
     "desktop",
     "router"
   ],
+  "editor.formatOnSave": false
```

Comment:
https://stackoverflow.com/questions/44831313/how-to-exclude-file-extensions-and-languages-from-format-on-save-in-vscode

let's do it just for the toml files

Reply:
corrected